assets/prometheus-adapter/deployment.yaml

```diff
@@ -26,21 +24,23 @@ spec:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
       labels:
         app.kubernetes.io/component: metrics-adapter
-        app.kubernetes.io/managed-by: cluster-monitoring-operator
         app.kubernetes.io/name: prometheus-adapter
         app.kubernetes.io/part-of: openshift-monitoring
         app.kubernetes.io/version: 0.8.4
     spec:
       affinity:
         podAntiAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-          - labelSelector:
-              matchLabels:
-                app.kubernetes.io/component: metrics-adapter
-                app.kubernetes.io/managed-by: cluster-monitoring-operator
-                app.kubernetes.io/name: prometheus-adapter
-                app.kubernetes.io/part-of: openshift-monitoring
-            topologyKey: kubernetes.io/hostname
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
The PR shouldn't change the affinity setting.

Reply:
The question came up before https://github.com/openshift/cluster-monitoring-operator/pull/1127/files/258fbefe14246acd3f7be65329181fa3ff5d9caf#r622147736

Iiuc this is the same change?

Reply:
Will wait for#1135 to merge, then rebase.

Reply:
oh good catch simon, I forgot that this PR would replace it. The easiest would be to rebase on top of #1135 and add the following code to the snippet:
```
    prometheusAdapter+: {
      podAntiAffinity: 'hard',
    },
```

Reply:
note that this change will allow removing the anti-affinity from prometheus-adapter.libsonnet: https://github.com/openshift/cluster-monitoring-operator/pull/1135/files#diff-ae9a6eb3e50ef89d556d7e9df8f1a13b29856ac7366161897e94b194dfde58fc

Reply:
it had merged :)

Reply:
:crossed_fingers: